### ISSUE TITLE
Adicionar novas entidades

### ISSUE THREAD

**Issue by Guilhermeslucas** [cacounicamp/Manual-do-Bixo#86]
Estou indo atrás do pessoal do Startup Lab. Falem aqui mais entidades que não estão no manual e deveriam estar que eu posso ver isso.

**Comment by Guilhermeslucas** [comment]
Vou atrás deles também!

**Comment by Guilhermeslucas** [comment]
Acabei de colocar o GER. Estou esperando o startup lab! #116

**Comment by VFerrari** [comment]
Pelo visto, nao houve resposta do Startup Lab. Mesmo assim, vou deixar a issue aberta, ate para o 018 ou se quisermos adicionar durante o ano.

**Comment by Guilhermeslucas** [comment]
Pessoal do Startup Lab me respondeu agora! Falei que não vai dar pra incluir pra esse ano! Vou mandar pull request, pelo menos já da pra fechar essa issue e aparecer no ano que vem!

**Comment by rmobis** [comment]
Bump! Rola incluir a UESL também?

**Comment by VFerrari** [comment]
Go nuts. Issue reaberto

**Comment by Guilhermeslucas** [comment]
Claro! Manda pra gente uma foto e descrição que eu incluo. Se quiser, pode
mandar o pull request também. O que for mais fácil.

Em 16/02/2017 14:20, "Raphael Mobis Tacla" <notifications@github.com>
escreveu:

**Comment by rmobis** [comment]
Pessoal, eu peguei pra fazer essa PR e acabei percebendo que tem muita coisa desatualizada na parte da AAACEC. To pensando em abrir uma issue pra organizar nosso pessoal nessas correções e já vou aproveitar e colocar a adição da UESL na lista da nova issue, pode ser?

**Comment by GoliasVictor** [comment]
Issue se mantem porque precisa atualizar as entidades e textos
